.github/workflows/haskell.yml

```diff
@@ -178,7 +179,7 @@ jobs:
           sed -i 's|tests: False|tests: True|g' cabal.project
         fi
 
-        cp .github/workflows/cabal.project.local.ci cabal.project.local
+        cp ".github/workflows/cabal.project.local.ci.$(uname -s)" cabal.project.local
```

Comment:
For Windows only, we need to pass extra flags so that `gcc` and `ld` (which `ghc` invokes) can find the libraries and header files.

This is necessary because unfortunately the 3rdparty `msys2` system we use installs packages into `D:/a/_temp/msys64/mingw64` instead of the system `C:\msys64\mingw64`.  This applies also the `secp256k1`.

It's unclear why `gcc` and `ld` invoked by `ghc` is unable to see this alternate installation directory.